curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-talk-about-hobbies-and-interests/657d9af10739f78d5401ad1d.md

```diff
@@ -0,0 +1,48 @@
+---
+id: 657d9af10739f78d5401ad1d
+title: Task 15
+challengeType: 19
+dashedName: task-15
+---
+
+# --description--
+
+The phrase `As a hobby` is used with the preposition `as`. Choose the correct preposition to complete the sentence.
+
+# --question--
+
+## --text--
+
+`Bob enjoys sketching and painting _ his hobby.`
+
+## --answers--
+
+`on`
+
+### --feedback--
+
+`On` is not the correct preposition to convey engagement in an activity.
+
+---
+
+`for`
+
+### --feedback--
+
+`For` implies doing something for a purpose, which is not the focus in this context.
+
+---
+
+`as`
+
+---
+
+`with`
+
+### --feedback--
+
+`With` does not convey the idea of pursuing an activity for personal enjoyment.
+
+## --video-solution--
+
+3
```

Comment:
The forth option is using "as" appropriately. I suggest:

`I use my phone as to make calls.`

### --feedback---

This sentence is incorrect because `as` is not needed with `to` in this context. The correct sentence should be: `I use my phone to make calls.`